<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">padding: 0px;
border: 0px;
background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
	background-color: white;
	border: 2px solid #faae2b;
	max-height: 50px;
	min-height: 50px;
	border-radius: 18px;
	margin: 0px;
}

QWidget:hover{
	background-color: white;
	border: 3px solid #faae2b;
	max-height: 50px;
	min-height: 50px;
	border-radius: 18px;
	margin: 0px;
}

QWidget:pressed{
	background-color: rgb(235, 235, 235);
	border: 3px solid #faae2b;
	max-height: 50px;
	min-height: 50px;
	border-radius: 18px;
	margin: 0px;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">max-height: 35px;
min-height: 35px;
border: 0px;
padding-left: 10px</string>
            </property>
            <property name="placeholderText">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: white;
	border: 2px solid transparent;
	max-height: 35px;
	min-height: 35px;
	max-width: 35px;
	min-width: 35px;
	border-radius: 18px;
	margin: 0px;
	padding: 0px;
}

QPushButton:hover{
	background-color: rgb(235, 235, 235);
	border: 3px solid transparent;
	max-height: 35px;
	min-height: 35px;
	max-width: 35px;
	min-width: 35px;
	border-radius: 18px;
	margin: 0px; 
	padding: 0px;
}

QPushButton:pressed{
	background-color: rgb(235, 235, 235);
	border: 4px solid transparent;
	max-height: 35px;
	min-height: 35px;
	max-width: 35px;
	min-width: 35px;
	border-radius: 18px;
	margin: 0px; 
	padding: 0px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Image/search_d.png</normaloff>../Image/search_d.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">border-radius: 20px;
background-color: #F4F4F4;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="listWidget_2">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QListWidget{
	border: 0px;
	margin-top: 0px;
}

QListWidget::item {
	background-color: white;
	height: 60px;
	margin: 5px;
	margin-bottom: 10px;
	border-radius: 15px;
	padding-left: 15px;
	padding-right: 15px;
}

QListWidget::item::hover{
	background-color: rgb(250, 250, 250);
	color: black;
	height: 60px;
	margin: 5px;
	margin-bottom: 10px;
	border-radius: 15px;
	padding-left: 15px;
	padding-right: 15px;
}

QListWidget::item::selected{
	background-color: rgb(250, 250, 250);
	color: black;
	height: 60px;
	margin: 5px;
	margin-bottom: 10px;
	border-radius: 15px;
	padding-left: 15px;
	padding-right: 15px;
	border: 2px solid gray;
}</string>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <item>
             <property name="text">
              <string>Abc</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DEFF</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Brehhh</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton_11">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	border: 1px solid black;
	height: 30px;
	border-radius: 10px;
	background-color: #FFCE41;
	color: white;
}

QPushButton::hover{
	border: 2px solid black;
	height: 30px;
	border-radius: 10px;
	background-color: #FFCE41;
}

QPushButton::pressed{
	border: 2px solid black;
	height: 30px;
	border-radius: 10px;
	background-color: rgb(234, 188, 59);
}</string>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../Image/new.png</normaloff>../Image/new.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	border: 1px solid black;
	height: 30px;
	border-radius: 10px;
	background-color: #FFCE41;
	color: white;
}

QPushButton::hover{
	border: 2px solid black;
	height: 30px;
	border-radius: 10px;
	background-color: #FFCE41;
}

QPushButton::pressed{
	border: 2px solid black;
	height: 30px;
	border-radius: 10px;
	background-color: rgb(234, 188, 59);
}</string>
               </property>
               <property name="text">
                <string> Remove</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../Image/remove.png</normaloff>../Image/remove.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	border: 1px solid black;
	height: 30px;
	border-radius: 10px;
	background-color: #FFCE41;
	color: white;
}

QPushButton::hover{
	border: 2px solid black;
	height: 30px;
	border-radius: 10px;
	background-color: #FFCE41;
}

QPushButton::pressed{
	border: 2px solid black;
	height: 30px;
	border-radius: 10px;
	background-color: rgb(234, 188, 59);
}</string>
               </property>
               <property name="text">
                <string>Edit</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../Image/all_note_view.png</normaloff>../Image/all_note_view.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
